ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.java

```diff
@@ -101,6 +101,32 @@ public void updateExtraData(ReactTextView view, Object extraData) {
       ReactAccessibilityDelegate.resetDelegate(
           view, view.isFocusable(), view.getImportantForAccessibility());
     }
+
+    CustomLineHeightSpan[] customLineHeightSpans =
+        spannable.getSpans(0, spannable.length(), CustomLineHeightSpan.class);
+    ReactAbsoluteSizeSpan[] absoluteSizeSpans =
+        spannable.getSpans(0, spannable.length(), ReactAbsoluteSizeSpan.class);
+    if (customLineHeightSpans.length > 0 && absoluteSizeSpans.length > 0) {
+      int highestLineHeight = 0;
+      for (CustomLineHeightSpan span : customLineHeightSpans) {
+        if (highestLineHeight == 0 || span.getLineHeight() > highestLineHeight) {
+          highestLineHeight = span.getLineHeight();
+        }
+      }
+
+      int highestFontSize = 0;
+      if (highestLineHeight != 0) {
+        for (ReactAbsoluteSizeSpan span : absoluteSizeSpans) {
+          if (highestFontSize == 0 || span.getSize() > highestFontSize) {
+            highestFontSize = span.getSize();
+          }
+        }
+
+        for (ReactAbsoluteSizeSpan span : absoluteSizeSpans) {
+          span.updateSpan(highestLineHeight, highestFontSize);
+        }
+      }
```

Comment:
run a loop over `absoluteSizeSpans` and check that any of them has textVerticalAlign different from center-child